lib/trino-collect/src/main/java/io/trino/collect/cache/EmptyCache.java

```diff
@@ -59,6 +61,18 @@ public V get(K key)
         return get(key, () -> loader.load(key));
     }
 
+    @Override
+    public ImmutableMap<K, V> getAll(Iterable<? extends K> keys)
+    {
+        try {
+            return loader.loadAll(keys).entrySet().stream()
+                    .collect(toImmutableMap(entry -> (K) entry.getKey(), Map.Entry::getValue));
```

Comment:
There is a warning in Intellij on this line.
Is it possible to avoid that?

Reply:
I've added `@SuppressWarnings("unchecked")`

Reply:
thank you for adding it!